cacticouncil/simcoast
# Packaging and Exporting

This PR should not in any way affect the actual gameplay of SimCoast. It is centered around making sure the game will run properly when it is exported to EXE format.

**How to test this PR**
As part of testing, I need everyone to do 2 things. However, these tests are only possible on a windows machine, so if you don't have one of those then I think you can just ignore it for now. 
1. Follow the exporting instructions in the wiki (link on sidebar) to create your own EXE and see if it runs. Make sure to do this on the Packaging branch. 
2. Extract the zip file I sent in _#developers_ in the Teams and see if that EXE works for you. It should work the same as the game you exported yourself in step 1.
